packages/components/src/components/collapsible/arrow-button.tsx

```diff
@@ -82,8 +83,16 @@ const ArrowButton = ({ is_collapsed = false, position, onClick, title }: TArrowB
             );
     }
 
+    if (flat_button) icon_arrow = <div className='dc-collapsible__icon__flat' data-testid='icon_flat' />;
```

Comment:
```suggestion
    if (handle_icon) icon_arrow = <div className='dc-collapsible__icon__handle' data-testid='handle_icon' />;
```
the correct name for this element is 'handle', please rename the classname and the prop )